src/pages/home.js

```diff
@@ -1,5 +1,29 @@
 import React from "react";
+import "./home.css";
+import logo from "./img/logo-img.png";
+import { Link } from "react-router-dom";
 
 export default function Home() {
-  return <h2>Home Page Test</h2>;
+  return (
+    <>
+      <section className="hero is-link is-fullheight-with-navbar">
+        <div className="hero-body body">
+          <div className="container">
+            <img src={logo} alt="Logo" class="logo-img" />
```

Comment:
```suggestion
            <img src={logo} alt="Logo" />
```

You could also consider using Bulma's image class (https://bulma.io/documentation/elements/image/) combined with the `is-rounded` class.